### PR TITLE
remove monorepo code/manifests/scaffolding

### PR DESCRIPTION
**Pull request by sdowell**
This change intends to remove all code, manifests, and scaffolding related to the monorepo mode. The monorepo code has been migrated and is managed separately from the multirepo code.

### CODE REVIEW

**Comment by sdowell** [comment]
Let me know if I missed anything 😄 
/assign @nan-yu 
/cc @rquitales 
/cc @xinnywinne

File: Makefile.build [hunk 1]
```diff
@@ -109,23 +93,12 @@ build-images-multirepo: license
 		.
 
 # Deprecated alias of build-images. Remove this once unused.
-.PHONY: image-nomos
-image-nomos: build-images
+.PHONY: build-images-multirepo
+build-images-multirepo: license
```
Comment (nan-yu) [inline Makefile.build:97]: license -> build-images

File: e2e/testcases/apiservice_test.go [hunk 2]
```diff
@@ -36,24 +35,12 @@ const (
 func TestCreateAPIServiceAndEndpointInTheSameCommit(t *testing.T) {
 	nt := nomostest.New(t, nomostesting.Reconciliation1, ntopts.Unstructured, ntopts.RequireGKE(t))
```
Comment (nan-yu) [inline e2e/testcases/apiservice_test.go:36]: @tiffanny29631 Remind me why do we require this test to only run on GKE cluster?

**Comment by sdowell** [comment]
Looks like I missed a few places, marking as WIP for now
/wip

**Review by nan-yu — COMMENTED** [review]
The following packages can also be pruned:
- pkg/policycontroller
- pkg/service
- pkg/syncer
- part of the `pkg/importer` package

**Comment by sdowell** [comment]
> The following packages can also be pruned:
> 
> * pkg/policycontroller
> * pkg/service
> * pkg/syncer
> * part of the `pkg/importer` package

@nan-yu Thank you for pointing that out! Looks like there's a few other places using things from `pkg/policycontroller` and a lot of places using things from `pkg/syncer`. I'll see how much I can clean up

**Comment by sdowell** [comment]
@nan-yu I took a pass at pruning the packages that you mentioned. Note that I only partially pruned `pkg/policycontroller`, `pkg/syncer`, and `pkg/importer` since there is shared code in these packages. PTAL and let me know if you think more can be pruned

File: pkg/policycontroller/constrainttemplate/controller.go [hunk 3]
```diff
@@ -42,24 +34,6 @@ var (
...
-	if err != nil {
-		return err
-	}
-
-	ct := EmptyConstraintTemplate()
-	return c.Watch(&source.Kind{Type: &ct}, &handler.EnqueueRequestForObject{})
-}
-
 // EmptyConstraintTemplate returns an empty ConstraintTemplate.
 func EmptyConstraintTemplate() unstructured.Unstructured {
```
[the remark this answers is not in the document]
Comment (sdowell) [reply pkg/policycontroller/constrainttemplate/controller.go:38]: Note there is also the `pkg/policycontroller/constants.go` file which is used by ~7 other places. But I agree and will move this function to `gatekeeper_test.go`

[the remark this answers is not in the document]
Comment (nan-yu) [reply pkg/policycontroller/constrainttemplate/controller.go]: `pkg/policycontroller/constants.go` only declares the `NamespaceSystem` constant. Sounds like that can be consolidated to some other place, maybe `pkg/syncer/differ/special_namespaces.go`?

[the remark this answers is not in the document]
Comment (sdowell) [reply pkg/policycontroller/constrainttemplate/controller.go]: I don't feel too strongly about it, although to me it seems fairly harmless to leave the constant defined in this package. I'm not sure that it makes more sense for the constant to be defined in the `differ` package from an API perspective

[the remark this answers is not in the document]
Comment (nan-yu) [reply pkg/policycontroller/constrainttemplate/controller.go]: I'm okay with keeping it in this package.

**Review by nan-yu — APPROVED** [review]
/lgtm

**Pull request merged**